### DISCUSSION TITLE
GUI Web Page not displaying

### DISCUSSION THREAD

**Discussion by TiloHarder** [Security-Onion-Solutions/securityonion#4929] [Unsupported Versions] [answered]
Hello everybody,

Trying to access the Security Onion GUI does not bring up the GUI Web page.
- Installation was fine
- sudo so-status worked fine
- sudo so-test worked fine
- sudo so-allow includes the whole subnet
- Telnet to the address with port 443 opens session
- with cleared browsercache browser warns about certificate
- if IP addresses used in browser, the browser translates the url to the hostname
but no GUI page shows up

**Comment by LWHaysUSDC** [comment]
@TiloHarder you may be running into the issue mentioned [here](https://github.com/Security-Onion-Solutions/securityonion/discussions/4718).   - Or not, if by "no GUI page shows up" you mean that the login page doesn't appear at all - in that case you've got something else.

The symptoms for  the above link are: you do get the SOC login screen but when you enter your (valid, so-allowed) username/password on the console web page it just redraws to blank with no error message.   This is in an environment with mixed-case hostnames (cookies) and it with browsers (chrome, chromium, IE, others) that do strict (case sensitive) hostname enforcement of cookies.   Firefox apparently (currently) does not so try firefox.  

In my case, I used a mixed case hostname/URL and the cookie got generated with all lowercase, and that was not good.  The workaround is to use firefox.   I actually fixed it by lowercasing everything and re-running so-setup on all the nodes.

**Reply by TiloHarder** [reply]
Thanks for your reply, but that was not the case.
There is no webpage showing up. No login screen. It just says server not found.
In Chrome as also in Firefox.

**Comment by TiloHarder** [comment, marked as answer]
Well, after a bit of additional testing i decided to install SecurityOnion again.
At the first attempt the installation stoped with an error message, but no errors in /root/errors.log.
At the second attempt the installation succeeded and this time the GUI is accessible.

The only difference i made is that i choose IP address instead of hostname for the access during setup questions.

**Reply by defensivedepth** [reply]
Yes, if you choose hostname for access, you will need to make sure that the hostname resolves to the IP address used by Security Onion.

**Reply by TiloHarder** [reply]
Well, ok!
But in my first scenario, when i choosed hostname, putting in the IP did also not bring up the page.
When putting in the IP the hostname showed up but the page did not appear.

**Reply by dougburks** [reply]
That's correct. When you choose the hostname option instead of the IP address option, we configure the web server such that any connections to the IP address will redirect to the hostname. So if you choose hostname, you must make sure that you are able to resolve the hostname using DNS or host entry.
